nautobot/dcim/tests/test_filters.py

```diff
@@ -775,70 +1293,162 @@ def test_manufacturer(self):
 
     def test_console_ports(self):
         params = {"console_ports": "true"}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 3)
         params = {"console_ports": "false"}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 0)
```

Comment:
This change from 2/1 to 3/0 gives me a bit of concern, as the 3 case is now indistinguishable from a "fail-open" unfiltered result, and the 0 case is now indistinguishable from a "fail-closed" unmatched result. Is there a way things could be adjusted in the common data (which I fully support!) such that these and other filters are still able to select a meaningful subset of the overall queryset?

Reply:
Agreed and fixed in 905b1482a